src/test/test_models.py

```diff
@@ -9,14 +9,14 @@ def test_merge_with_bounding_boxes_and_pages():
         id="1",
         text="Hello",
         chunk_type=BlockType.TEXT,
-        bounding_boxes=[[(0, 0), (1, 1)]],
+        bounding_boxes=[[(0, 0), (1, 1), (1, 0), (0, 1)]],
         pages=[1],
     )
     chunk2 = Chunk(
         id="2",
         text="World",
         chunk_type=BlockType.TEXT,
-        bounding_boxes=[[(2, 2), (3, 3)]],
+        bounding_boxes=[[(2, 2), (3, 3), (3, 2), (2, 3)]],
```

Comment:
do we care about validating the order of these points? 

as far as i can tell this bounding box is set up as `[(Xmin, Ymin), (Xmax, Ymax), (Xmax, Ymin), (Xmin, Ymax)]` (top-left, bottom-right, bottom-right, top-left)

but the front end is looking for `[(Xmin, Ymin), (Xmax, Ymin), (Xmax, Ymax), (Xmin, Ymax)]`

Reply:
good idea! have implemented that and requested review again